tests/integration/run-cli.js

```diff
@@ -37,7 +37,11 @@ function runCliWorker(dir, args, options) {
 
   const worker = new Worker(CLI_WORKER_FILE, {
     argv: args,
-    execArgv: ["--trace-deprecation"],
+    execArgv: [
+      "--trace-deprecation",
+      "--disable-warning=ExperimentalWarning",
+      "--experimental-strip-types",
```

Comment:
Would this work in Node < 22.6 where this [flag was introduced](https://nodejs.org/en/blog/release/v22.6.0)? We should make sure we don’t get any warnings in all supported Node versions and if the flag is not available, there should be a meaningful error about why `prettier.config.ts` could not load. Docs need to reflect min version for TS config too, unless we want to change the approach and use a built-in transpiler.

Reply:
It should be passed via `options` and only set when testing ts configs.